### PR TITLE
Feature/1394 login

### LINKED ISSUE #1394: Polish the current login screen and login experience
**Is your feature request related to a problem? Please describe.**
The `/loginForm` page is not styled and looks pretty awful.

From #1260, the logout link should be clearer between guests and logged in users.

From #660, admins will have a polished way to login.

**Describe the solution you'd like**
The `/loginForm` page is styled.

**Additional context**
We'd like to change behavior around logging in, guests, and browsing civiform programs. But that may be more work than we have time to do, so this is just a polish task.

**Done when**
The `/loginForm` page is styled.

### PR DESCRIPTION
**Pull request by kendrickt**
### Description
Style up the login page. There should be no functional changes. The debug admin login options just show up on the side. 

Button mapping:
* `Login with IDCS` -> `LOG IN`
* `Login with ADFS` -> `Admin login`
* `Register` -> `CREATE AN ACCOUNT` (hidden in staging because `config.hasPath("idcs.register_uri")` is false)
* `Continue` -> `CONTINUE AS GUEST`

### old:
image

### figma:
image

### new
image

### new mobile
image

### Issue(s)
Fixes #1394

### CODE REVIEW

File: universal-application-tool-0.0.1/app/views/LoginForm.java [hunk 1]
```diff
@@ -28,55 +35,134 @@ public LoginForm(BaseHtmlLayout layout, Config config) {
...
+            redirectButton(
+                "program-admin",
+                "Of All Active Programs",
+                routes.CallbackController.fakeAdmin(
+                        FakeAdminClient.CLIENT_NAME, FakeAdminClient.PROGRAM_ADMIN)
+                    .url()));
+  }
+
+  private Tag loginButton() {
+    String msg = "LOG IN";
```
Comment (natsid) [inline universal-application-tool-0.0.1/app/views/LoginForm.java:143]: suggestion (blocking): In general, these should be written in Sentence case ("Log in", "Create account", "Continue as guest"). There is style that transforms them to upper case (https://tailwindcss.com/docs/text-transform), which I think we already use in our base applicant styles.

File: universal-application-tool-0.0.1/app/views/LoginForm.java [hunk 2]
```diff
@@ -28,55 +35,134 @@ public LoginForm(BaseHtmlLayout layout, Config config) {
...
+    return redirectButton(
+            "idcs", msg, routes.LoginController.idcsLoginWithRedirect(Optional.empty()).url())
+        .withClasses(BaseStyles.LOGIN_REDIRECT_BUTTON);
+  }
+
+  private Tag createAccountButton() {
+    String msg = "CREATE ACCOUNT";
```
Comment (natsid) [inline universal-application-tool-0.0.1/app/views/LoginForm.java:150]: question (blocking): Should we register these in our messages file? I know they probably won't have selected a language yet, but I know we put some strings in `conf/messages` and not in the translations files when they are only used before a language is selected.

Comment (natsid) [reply universal-application-tool-0.0.1/app/views/LoginForm.java:150]: Not sure if we gain anything by doing that tho. Only do if minor.

File: universal-application-tool-0.0.1/app/views/LoginForm.java (hunk 1 above)
Comment (kendrickt) [reply universal-application-tool-0.0.1/app/views/LoginForm.java:143]: done!

File: universal-application-tool-0.0.1/app/views/LoginForm.java (hunk 2 above)
Comment (kendrickt) [reply universal-application-tool-0.0.1/app/views/LoginForm.java:150]: done!

**Review by natsid — APPROVED** [review]

**Pull request merged**